src/main/resources/messages_de.properties

```diff
@@ -0,0 +1,37 @@
+downl_check_database=Lade Prüfungsdatenbank {0} herunter
+finish_download_check_database=Erfolgreich Prüfungsdatenbank {0} heruntergeladen
+reloading_local_check_database=Lade lokale Prüfungsdatenbank neu.
+finish_reload_local_check_database=Erfolgreich Prüfungsdatenbank neugeladen.
+rescan_updated=Neuanalysieren aufgrund von einer aktualisierten Datenbank.
+finish_rescan=Neuanalysieren fertiggestellt.
+downl_checksum_database=Lade Prüfsumme Datenbank {0} herunter.
+finish_download_checksum_database=Erfolgreich Prüfsumme Datenbank {0} heruntergeladen.
+reloading_checksum_database=Lade lokale Prüfsumme Datenbank neu.
+reloaded_checksum_database=Erfolgreich lokale Prüfsumme Datenbank neugeladen.
+not_watching=Nicht anschauen {0}.
+interrupted_thread=Ordner Beobachter Schraubengewinde unterbrochen.
+not_recognized=Anschauen Schlüssel nicht wiedererkannt.
+dir_inaccessible={0} ist nicht Betrittbar. Stoppe Armbanduhr.
+not_in_database={0} war nicht in der Prüfsumme Datenbank, also wird es neu analysiert für heimtückischen Code.
+checking=Überprüfe, ob {0} mit {1} ({2}) angesteckt ist.
+might_not_be_infected={0} ist VIELLEICHT nicht angesteckt mit {1} ({2}) 
```

Comment:
`infiziert` is a better word for that. `Angesteckt` is only used using biomass e.g. humans, animals